pkg/vcs/linux.go

```diff
@@ -8,19 +8,26 @@ import (
 	"io"
 	"net/mail"
 	"path/filepath"
+	"regexp"
 	"sort"
 	"strconv"
 	"strings"
 	"time"
 
-	"github.com/google/syzkaller/pkg/email"
 	"github.com/google/syzkaller/pkg/osutil"
 )
 
 type linux struct {
 	*git
 }
 
+type RecipientInfo struct {
+	Email         string
+	RealName      string
```

Comment:
These 2 fields are better expressed as `net/mail.Address`. It contains these 2 fields with names according RFC and some useful helper functions.